google/cloud/opentelemetry/internal/time_series.cc

```diff
@@ -215,6 +217,7 @@ std::vector<google::monitoring::v3::TimeSeries> ToTimeSeries(
       }
     }
   }
+  WithExtraLabels(data, tss);
```

Comment:
I think it would make sense to build this into `ToMetric(...)` which deals specifically with the `google.api.Metric`. It also has logic for changing strings from `service.name` -> `service_name` which we can reuse.